.github/workflows/pr-check.yml

```diff
@@ -0,0 +1,169 @@
+        run: |
+          echo running on PR ${GITHUB_REF}, branch - ${GITHUB_HEAD_REF}
+          echo "pr_number=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')" >> "$GITHUB_ENV"
+
+      - name: Configuring nodejs 16.x version
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Pull Che-Code Docker image
+        run: |
+          #
+          # pull che-code:latest docker image
+          #
+          docker pull quay.io/che-incubator/che-code:latest
+
+      - name: Start minikube
+        id: run-minikube
+        uses: che-incubator/setup-minikube-action@next
+        with:
+          minikube-version: v1.23.2
+
+      - name: Install chectl
+        run: bash <(curl -sL https://www.eclipse.org/che/chectl/) --channel=next
+
+      - name: Deploy Che
+        run: |
+          #
+          # load Che-Code image into minikube
+          #
+          minikube image load quay.io/che-incubator/che-code:latest
+          minikube image list
+
+          # get patch file to set up resources
+          wget https://raw.githubusercontent.com/che-incubator/che-code/main/build/test/github-minikube-checluster-patch.yaml -P /tmp
+          
+          #
+          # deploy Che
+          #
+          chectl server:deploy \
+            --batch \
+            --platform minikube \
+            --k8spodwaittimeout=120000 \
+            --k8spodreadytimeout=120000 \
+            --che-operator-cr-patch-yaml "/tmp/github-minikube-checluster-patch.yaml"
+
+          #
+          # apply patch
+          #
+          kubectl patch devworkspaceoperatorconfigs \
+            -n eclipse-che devworkspace-config \
+            --patch '{"config": {"workspace": {"imagePullPolicy": "IfNotPresent"}}}' \
+            --type merge
+
+      - name: Pull Universal Base Image
+        run: |
+          minikube image pull quay.io/devfile/universal-developer-image:ubi8-latest
+
+      - name: Run Empty Workspace API test
+        run: |
+          cd tests/e2e
+          npm ci
+          export TS_PLATFORM=kubernetes && 
+          export TS_API_TEST_KUBERNETES_COMMAND_LINE_TOOL=kubectl && 
+          export TS_SELENIUM_VALUE_OPENSHIFT_OAUTH=false && 
+          export TS_SELENIUM_BASE_URL=https://$(kubectl get ingress che -n eclipse-che -o jsonpath='{.spec.rules[0].host}') && 
+          export NODE_TLS_REJECT_UNAUTHORIZED=0 && 
+          export TS_SELENIUM_LOG_LEVEL=TRACE && 
+          export TS_SELENIUM_DEFAULT_ATTEMPTS=2 &&
+          export USERSTORY=EmptyWorkspaceAPI && 
+          export TS_API_TEST_UDI_IMAGE=quay.io/devfile/universal-developer-image:ubi8-latest &&
+          npm run driver-less-test
+
+      - name: Build E2E test docker image
+        run: |
+          set -xe
+
+          cd tests/e2e
+          docker build -t quay.io/eclipse/che-e2e:"${{ env.pr_number }}" -f build/dockerfiles/Dockerfile .
+
+      - name: Run Empty Workspace UI test from che-e2e container
```

Comment:
IMHO, it make sense to run SmokeTest instead - it covers project import additionally https://github.com/eclipse/che/blob/main/tests/e2e/specs/SmokeTest.spec.ts

Also, there is a plan to run SmokeTest in PR check of Eclipse Che repos:
- https://github.com/eclipse/che/issues/22046
- https://github.com/eclipse/che/issues/22045
- https://github.com/eclipse/che/issues/22047
- https://github.com/eclipse/che/issues/22101

Reply:
Please, read my description

Reply:
+ we check that changes not broke e2e test functionality, commands from packege.json. We don`t check "che" here.

Reply:
https://github.com/eclipse/che/pull/22377#issuecomment-1649304686

Reply:
Added